x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.ts

```diff
@@ -166,6 +171,7 @@ export function registerTransactionDurationRuleType({
                     missing: ENVIRONMENT_NOT_DEFINED.value,
                   },
                   { field: TRANSACTION_TYPE },
+                  { field: TRANSACTION_NAME },
```

Comment:
`transaction.name` is a high cardinality field. Not sure if we should group by it. What are the consequences if we do?

Reply:
https://github.com/elastic/kibana/issues/152329#issuecomment-1497345002